README.md

```diff
@@ -120,7 +120,12 @@ bundle exec bin/rails r -e production "BadIso8601Dates.report" > BadIso8601Dates
 scp deploy@sdr-infra:/opt/app/deploy/dor-services-app/BadIso8601Dates.csv BadIso8601Dates.csv
 ```
 
-## Releasing
+## Releasing a patch change
+A patch change is a change that (1) does affect the data model; (2) does not alter the openapi.yml; and more broadly (3) does not matter if some applications have the change and others do not.
```

Comment:
Should `(1)` here read `does not affect the data model`?

Reply:
And that is why you have code reviews ...